app/i18n/de/conf.php

```diff
@@ -60,7 +60,7 @@
 		'api' => 'API-Verwaltung',
 		'delete' => array(
 			'_' => 'Accountlöschung',
-			'warn' => 'Dein Account und alle damit bezogenen Daten werden gelöscht.',
+			'warn' => 'Ihr Account und alle damit bezogenen Daten werden gelöscht.',
```

Comment:
Presumably you could alternatively say `dieses Account`? If that's preferable.

Reply:
It would be also fine to make it more clear and less personal to "Dieser Account"